examples/gitlab/README.md

```diff
@@ -0,0 +1,46 @@
+## GitLab Push EventListener
+
+Creates an EventListener that listens for Gitlab webhook events. To expose the
+EventListener, add a `serviceType: LoadBalancer`.
+
+### Try it out locally:
+
+1. Create the service account:
```

Comment:
I think the numbers are off in this list. Maybe turn them into subheaders instead of a list?
`#### 1. Create the ServiceAccount`

Reply:
fixed!